pkg/crc/validation/validation.go

```diff
@@ -27,15 +27,15 @@ func ValidateDriver(driver string) error {
 // ValidateCPUs checks if provided cpus count is valid
 func ValidateCPUs(value int) error {
 	if value < constants.DefaultCPUs {
-		return errors.Newf("CPUs required >=%d", constants.DefaultCPUs)
+		return errors.Newf("required CPUs >= %d", constants.DefaultCPUs)
```

Comment:
`require ` (to be)

Reply:
Here's how messages from the Validate* function fit into the complete error message:

$ crc config set memory 100
ERRO required memory in MiB >= 8192               
Value '100' for configuration property 'memory' is invalid, reason: required memory in MiB >= 8192
$

Require doesn't seem quite right for this sentence...

Reply:
`Requires memory in MiB to be >= 8192`